mscore/instrwidget.h

```diff
@@ -31,14 +31,15 @@ class InstrumentGenre;
 
 enum class ListItemOp : char { KEEP, I_DELETE, ADD, UPDATE };
 enum { PART_LIST_ITEM = QTreeWidgetItem::UserType, STAFF_LIST_ITEM };
+enum class PartiturSelectionType : char { PARENTS_SELECTED, CHILDREN_SELECTED, NONE_SELECTED };
 
 //---------------------------------------------------------
 //   PartListItem
 //---------------------------------------------------------
 
 class PartListItem : public QTreeWidgetItem {
 
-   public:
+public:
```

Comment:
All class scoping rules should be 3 spaces from the start of the class indentation (in this case 0 + 3 = 3 spaces). Make sure to undo this change and the following instances in this file.

Reply:
Going to revert it. I applied to coding styles definition from the handbook in Qt Creator, but apparently this cannot be controlled correctly.